cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/engine/sql/SQLEngine.java

```diff
@@ -110,4 +107,13 @@
    * @param datasetName boolean specifying if all running tasks should be stopped at this time (if any are running).
    */
   void cleanup(String datasetName) throws SQLEngineException;
+
+  /**
+   * Builds a new stage name for the SQL Engine implementation.
+   * @param name the name of the plugin to use
+   * @return new name for the SQL engine stage.
+   */
+  static String buildStageName(String name) {
```

Comment:
doesn't look like something that should be in the API class, but something in some internal class